d/Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /usr/src/app
 {{/files}}
 
 ENV DC=ldc2
-RUN dub build -b release --compiler=ldc2 --verbose
+RUN dub build -b release-nobounds --compiler=ldc2 --verbose
```

Comment:
why ?

Reply:
I've checked other sources - for example Nim - and they disable bounds check as well. It could help with performance. Also for better performance I need to add -flto-full but it needs be tested before adding

Reply:
Do whatever is recommended / best-practice for this language

Reply:
I recommend never using even release since it can remove useful assertions, and release-nobounds is just plain irresponsible for a real world deployment.

Only benchmarks and fools use these things.

Reply:
so we could probably also remove it for `nim` @cyrusmsk (probably with an other PR btw)

Reply:
Ok, I removed nobounds flag in last commit